README.md

```diff
@@ -10,21 +10,40 @@
 [![Docker Image Size (latest by date)](https://img.shields.io/docker/image-size/cisagov/example)](https://hub.docker.com/r/cisagov/example)
 [![Platforms](https://img.shields.io/badge/platforms-amd64%20%7C%20arm%2Fv6%20%7C%20arm%2Fv7%20%7C%20arm64%20%7C%20ppc64le%20%7C%20s390x-blue)](https://hub.docker.com/r/cisagov/pca-report-generator-docker/tags)
 
-This is a Docker skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) GitHub Docker project
-started.  This skeleton project contains [licensing
-information](LICENSE), as well as [pre-commit hooks](https://pre-commit.com)
-and [GitHub Actions](https://github.com/features/actions) configurations
-appropriate for Docker containers and the major languages that we use.
+This is a Docker project that containerizes the [pca-report-library](https://github.com/cisagov/pca-report-library)
+package, which can be used to generate Phishing Campaign Assessment (PCA) reports.
 
 ## Running ##
 
-### Running with Docker ###
+The following Docker commands are available.
 
-To run the `cisagov/example` image via Docker:
+An alias can also be set beforehand to remove redundancy.
+
+Parameters can be added at the end. Each defaulting command prints out a help message.
+
+`pca-report-generator` - Creates a PCA report as a PDF
+
+```console
+docker run -v $(pwd):/home/cisa cisagov/pca-report-generator:0.0.1
+```
+
+`pca-report-templates` - Exports the Report Mustache template and Manual data
+file template
+
+```console
+docker run -v $(pwd):/home/cisa --entrypoint pca-report-templates cisagov/pca-report-library-docker:0.0.1
+```
```

Comment:
Improve and correct examples:
````suggestion
`pca-report-generator` - Create a PCA report as a PDF:

```console
docker run --volume $(pwd):/home/cisa cisagov/pca-report-library:0.0.1 MY_ASSESSMENT_ID
```

`pca-report-templates` - Export the PCA manual data file template or Mustache
template:

```console
docker run --volume $(pwd):/home/cisa --entrypoint pca-report-templates cisagov/pca-report-library:0.0.1 --manualData

docker run --volume $(pwd):/home/cisa --entrypoint pca-report-templates cisagov/pca-report-library:0.0.1 --LaTeX
```
````